planetiler-core/src/main/java/com/onthegomap/planetiler/FeatureMerge.java

```diff
@@ -216,10 +216,23 @@ public static List<VectorTile.Feature> mergeLineStrings(List<VectorTile.Feature>
             }
           }
         }
+        
+        merger = new LoopLineMerger();
+        for (var outputSegment : outputSegments) {
+          merger.add(outputSegment);
+        }
+        outputSegments = merger.getMergedByAngle();
+
```

Comment:
it would be nice not to need to need a separate step, can we make the angle merging part of the loop-line-merger path - maybe add a configuration option like "merger.mergeByAngle(true)" ?

Reply:
I think we need to do it after line simplification. But I agree it would be better not having to build the graph twice. Should we simplify in the looplinemerger class?